### PR TITLE
chore: added player

### PR DESCRIPTION
**Pull request by hrishikeshpaul**

### CODE REVIEW

File: vb-be/src/modules/player/player.controller.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { Controller, Put, Response, Query, Headers } from '@nestjs/common';
+import { Response as Res } from 'express';
+
+import { HttpStatus } from 'src/util/http';
+import { ErrorHandler, ErrorText } from 'src/util/error';
+
+import { TagService } from '@modules/tag/tag.service';
+import { PlayerService } from './player.service';
```
Comment (s-munro) [inline vb-be/src/modules/player/player.controller.ts:8]: nit: imports

File: vb-be/src/modules/player/player.module.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { HttpModule } from '@nestjs/axios';
+import { Module } from '@nestjs/common';
+import { PlayerController } from './player.controller';
+import { PlayerService } from './player.service';
```
Comment (s-munro) [inline vb-be/src/modules/player/player.module.ts:4]: same here

File: vb-fe/src/App.tsx [hunk 3]
```diff
@@ -14,6 +14,10 @@ import "App.scss";
 import { useSocket } from "core/socket/useSocket";
 import { RoomForm } from "util/Room";
 import { createRoomAction } from "_store/room/RoomActions";
+import { usePlayer } from "core/player/usePlayer";
+import { Button } from "@chakra-ui/react";
```
Comment (s-munro) [inline vb-fe/src/App.tsx:18]: nit: imports

File: vb-fe/src/_store/player/PlayerReducer.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import { PlayerConstants, PlayerActionTypes } from "_store/player/PlayerTypes";
+
+// import { Error } from "app/models/system.model";
```
Comment (s-munro) [inline vb-fe/src/_store/player/PlayerReducer.ts:3]: unused comment

File: vb-fe/src/core/player/usePlayer.ts [hunk 5]
```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from "react";
+import { useDispatch } from "react-redux";
+import { PlayerConstants } from "_store/player/PlayerTypes";
```
Comment (s-munro) [inline vb-fe/src/core/player/usePlayer.ts:3]: nit: import order

File: vb-fe/src/core/player/usePlayer.ts [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        volume: 0.2,
+      });
+
+      webPlayer.addListener("ready", (data: any) => {
+        console.log("Ready with Device ID", data);
+        dispatch({
+          type: PlayerConstants.ADD_DEVICE,
+          payload: data.device_id,
+        });
+        // play("spotify:album:0Nu5uWgrnNGWWbWfzXPgPI", data.device_id);
```
Comment (s-munro) [inline vb-fe/src/core/player/usePlayer.ts:43]: comment here

**Review by s-munro — CHANGES_REQUESTED** [review]
Everything is looking pretty solid at first glance, just a bunch of small things relating to imports and comments.  Going to checkout to this branch and test it out in a moment

**Comment by hrishikeshpaul** [comment]
closing this -- added this in this in the branch `feat/hp/room2`

**Pull request closed without merging**